server/internal/oteltest/server_otel_test.go

```diff
@@ -97,7 +97,11 @@ func TestOTelTracing(t *testing.T) {
 					attribute.String("http.method", httpMethod),
 					attribute.String("http.route", helloRouteName),
 				},
-				expectedOpNameHelloHTTPSpan: expectedAttrsHelloHTTPSpan,
+				expectedOpNameHelloHTTPSpan: {
+					attribute.String("http.target", helloRouteURL.Path),
+					attribute.String("http.method", httpMethod),
+					attribute.String("http.route", helloRouteName),
+				},
```

Comment:
I think that if HTTP changes above, we should also update httpgrpc tracing to match the attributes.